### PR TITLE
rpm-packaging: Support multiple release updates

### PR DESCRIPTION
**Pull request by toabctl**
There is now a stable/mitaka branch for the rpm-packaging project.
For the CI, we need to update the available packages for all branches
(now stable/mitaka and master). We use the subprojects in OBS under
Cloud:OpenStack:Upstream for that.

### CODE REVIEW

File: scripts/jenkins/jobs-obs/templates/openstack-rpm-packaging-update.yaml [hunk 1]
```diff
@@ -14,7 +14,7 @@
           #!/bin/bash
           export OSCAPI="https://api.opensuse.org"
           export JHOME="/home/jenkins"
-          export OBS_PROJECT="Cloud:OpenStack:Upstream"
+          export OBS_PROJECT="Cloud:OpenStack:Upstream:{release}"
```
Comment (aplanas) [inline scripts/jenkins/jobs-obs/templates/openstack-rpm-packaging-update.yaml]: missing a $?

**Comment by aplanas** [comment]
LGTM +1

**Comment by toabctl** [comment]
@aplanas added another commit...

**Comment by dirkmueller** [comment]
need this for fixing test suite:

--- a/Makefile
+++ b/Makefile
@@ -44,7 +44,7 @@ python_unittest:
 
 jjb_test:
        jenkins-jobs --ignore-cache test scripts/jenkins/jobs-ibs:scripts/jenkins/jobs-ibs/templates/ cloud* openstack* > /dev/null
-       jenkins-jobs --ignore-cache test scripts/jenkins/jobs-obs cloud* openstack* > /dev/null
+       jenkins-jobs --ignore-cache test scripts/jenkins/jobs-obs:scripts/jenkins/jobs-obs/templates/ cloud* openstack* > /dev/null

**Comment by dirkmueller** [comment]
@toabctl would be good to add the zuul config changes to the related git commits (when its working)

**Comment by toabctl** [comment]
> @toabctl would be good to add the zuul config changes to the related git commits (when its working)

Done

**Comment by dirkmueller** [comment]
+1

**Pull request merged**